deployment_scripts/linode-marketplace-harbor/harbor-deploy.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+set -e
+trap "cleanup $? $LINENO" EXIT
+
+##Linode/SSH security settings
+#<UDF name="user_name" label="The limited sudo user to be created for the Linode" default="">
+#<UDF name="password" label="The password for the limited sudo user" example="an0th3r_s3cure_p4ssw0rd" default="">
+#<UDF name="disable_root" label="Disable root access over SSH?" oneOf="Yes,No" default="No">
+#<UDF name="pubkey" label="The SSH Public Key that will be used to access the Linode (Recommended)" default="">
+
+## Domain Settings
+#<UDF name="token_password" label="Your Linode API token. This is needed to create your WordPress server's DNS records" default="">
+#<UDF name="subdomain" label="Subdomain" example="The subdomain for the DNS record: www (Requires Domain)" default="">
+#<UDF name="domain" label="Domain" example="The domain for the DNS record: example.com (Requires API token)" default="">
+
+## harbor Settings 
+#<UDF name="harbor_admin_password" label="The Harbor admin password">
```

Comment:
Can we generate a password for this in the playbook and put it in /root/credentials? You can copy what we do for the mysql helper

https://github.com/akamai-compute-marketplace/marketplace-apps/blob/main/apps/linode_helpers/roles/securemysql/tasks/passgen.yml#L6